certified-connectors/LAWLIFT/apiProperties.json

```diff
@@ -0,0 +1,25 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "You can acquire the key at app.lawlift.com",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#000000",
```

Comment:
@makaveli1313 Please make sure that the background of your icon is `#000000`

Reply:
> @makaveli1313 Please remove icon from the PR

Thank you for your answers. We should move it to our folder? It's not clear from the documentation.